include/lsqecc/patches/patches.hpp

```diff
@@ -62,7 +62,8 @@ enum class PatchActivity : uint8_t
     Unitary,
     Distillation,
     Dead,
-    Busy
+    BusyClearNextSetp,
+    BusyDontClear
```

Comment:
Tried to make the usage of busy explicit, but it seems like there are none at this point.

Looks like we can remove these two and `Busy` altogether.

Reply:
I think removing all the "Busy"s sounds like a great idea, if it can be done without breaking anything. Ideally, I'd like to see exactly what is happening at each patch in each slice.

One thing to bear in mind when making these changes is that we should be *very* careful to ensure all preconditions are always checked when calling the `try_apply_instruction` functions. We had some really hard to detect bugs caused by subtle issues with precondition checking. These preconditions might change as PatchActivity cases are added and removed.